src/app/_common/actions/userPreferencesActions.js

```diff
@@ -39,3 +43,14 @@ export async function removeClosedFilter(panelId) {
     const newCookieValue = { closedFilters: closedFilters };
     cookies().set(USER_PREFERENCES_COOKIE_NAME, JSON.stringify(newCookieValue), COOKIE_OPTIONS);
 }
+
+export async function dismissPanel(panelId) {
+    if (!ALLOWED_DISMISSED_PANELS.includes(panelId)) {
+        // Trying to set an invalid value
+        return;
+    }
+    const existingCookie = await getUserPreferences();
+    const dismissedPanels = new Set(existingCookie.dismissedPanels || []).add(panelId);
+    const newCookieValue = { dismissedPanels: [...dismissedPanels] };
```

Comment:
Bør vi bruke  { ...existingCookie, dismissedPanels: [...dismissedPanels]}  for å ikke miste verdien på closedFilters? Vi må evt gjøre det samme på addClosedFilter() og removeClosedFilter()

Reply:
Bra du så den :-)